flink-runtime/src/main/java/org/apache/flink/runtime/throughput/ThroughputCalculator.java

```diff
@@ -71,4 +71,8 @@ public long calculateThroughput() {
 
         return lastThroughput;
     }
+
+    public long lastThroughput() {
```

Comment:
nit: `getLastThroughput()`. Is this being used anywhere?

Reply:
good catch. Initially, I refactored in such a way that I would needed this getter but then I changed my mind but forgot to remove it.